src/platforms/android/androidappimageprovider.h

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef ANDROIDAPPIMAGEPROVIDER_H
+#define ANDROIDAPPIMAGEPROVIDER_H
+
+#include <QObject>
+#include <QQuickImageProvider>
+#include <QAndroidJniObject>
+
+class AndroidAppImageProvider : public QQuickImageProvider, public QObject {
```

Comment:
final?

Reply:
Do you still need QObjct if you have QQuickImageProvider?

Reply:
QQuickImageProvider is not a QObject :(